src/EnergyPlus/OutputReportTabularAnnual.cc

```diff
@@ -1353,7 +1358,9 @@ std::vector<Real64> AnnualTable::calculateBins(int const numberOfBins,
         } else {
             // determine which bin the results are in
             binNum = int((*valueIt - bottomOfBins) / intervalSize);
-            returnBins[binNum] += *elapsedTimeIt;
+            if (binNum < binNum && binNum >= 0) {
```

Comment:
Hmmmmm

```
if (binNum < binNum && binNum >= 0) {
```

That first comparison doesn't look right....

Reply:
It seems like it should be binNum < numberOfBins. But my question would be what does in mean when the binNum calculation comes out < 0 or > numberOfBins? Doesn't that mean this calculation should be in returnBins[0] or returnBins[numberOfBins], respectively? Or maybe it means that this calculation is outside the bin limits and should not be counted.

Reply:
"Hmmmmmm" is right, what was I thinkin' 

It has been a while since I looked at this but I better take another look at what I was trying to do.